docs/operators/remove.md

```diff
@@ -0,0 +1,343 @@
+## `remove` operator
+
+The `remove` operator removes a field from a record.
+
+It's configured by passing a field to remove.
+
+### Configuration Fields
+
+| Field      | Default          | Description                                                                                                                                                                                                                              |
+| ---        | ---              | ---                                                                                                                                                                                                                                      |
+| `id`       | `restructure`    | A unique identifier for the operator                                                                                                                                                                                                     |
+| `output`   | Next in pipeline | The connected operator(s) that will receive all outbound entries                                                                                                                                                                         |
+| `field`      | required       | The [field](/docs/types/field.md) to remove.
+| `on_error` | `send`           | The behavior of the operator if it encounters an error. See [on_error](/docs/types/on_error.md)                                                                                                                                          |
+| `if`       |                  | An [expression](/docs/types/expression.md) that, when set, will be evaluated to determine whether this operator should be used for the given entry. This allows you to do easy conditional parsing without branching logic with routers. |
+
+Example usage:
+
+<hr>
+
+Remove value from body
+```yaml
+- type: remove 
+    field: 
+    - key1
```

Comment:
Is this right? Does it accept an array?

I think we probably just want one field here. It's easy enough to chain instances of the operator together if needed.